diff --git a/php/modules/Album.php b/php/modules/Album.php
index ab4a545..5978ebc 100644
--- a/php/modules/Album.php
+++ b/php/modules/Album.php
@@ -323,7 +323,11 @@ class Album extends Module {
 
 			# Add to array
 			$files[] = $zipFileName;
-
+			if (isset($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
+				if (preg_match('/^ja/i', $_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
+					 $zipFileName = mb_convert_encoding($zipFileName, 'sjis-win', 'UTF-8');
+				}
+			}			
 			# Add photo to zip
 			$zip->addFile($photo->url, $zipFileName);
 
@@ -559,4 +563,4 @@ class Album extends Module {
 
 }
 
-?>
\ No newline at end of file
+?>
