--- php.ini.old	2014-05-08 15:28:50.925968235 +0900
+++ php.ini	2014-05-08 15:30:43.075968236 +0900
@@ -372,7 +372,7 @@
 ; Maximum execution time of each script, in seconds
 ; http://php.net/max-execution-time
 ; Note: This directive is hardcoded to 0 for the CLI SAPI
-max_execution_time = 30
+max_execution_time = 200
 
 ; Maximum amount of time each script may spend parsing request data. It's a good
 ; idea to limit this time on productions servers in order to eliminate unexpectedly
@@ -660,7 +660,7 @@
 ; Its value may be 0 to disable the limit. It is ignored if POST data reading
 ; is disabled through enable_post_data_reading.
 ; http://php.net/post-max-size
-post_max_size = 8M
+post_max_size = 200M
 
 ; Automatically add files before PHP document.
 ; http://php.net/auto-prepend-file
@@ -788,10 +788,10 @@
 
 ; Maximum allowed size for uploaded files.
 ; http://php.net/upload-max-filesize
-upload_max_filesize = 2M
+upload_max_filesize = 20M
 
 ; Maximum number of files that can be uploaded via a single request
-max_file_uploads = 20
+max_file_uploads = 100
 
 ;;;;;;;;;;;;;;;;;;
 ; Fopen wrappers ;
