--- config.inc.php.new	2014-04-12 00:22:04.000000000 +0900
+++ config.inc.php	2014-04-16 16:31:20.251600026 +0900
@@ -8,18 +8,18 @@
  * @license    http://www.wtfpl.net/ see COPYING file
  */
 
-define ('SALT', ''); # put a strong string here
+define ('SALT', 'f8cf4bb11dbc63285871fff098e4aec2'); # put a strong string here
 define ('LANG', 'en_EN.utf8');
 
-define ('STORAGE', 'sqlite'); # postgres, mysql or sqlite
+define ('STORAGE', 'mysql'); # postgres, mysql or sqlite
 
 define ('STORAGE_SQLITE', ROOT . '/db/poche.sqlite'); # if you are using sqlite, where the database file is located
 
 # only for postgres & mysql
 define ('STORAGE_SERVER', 'localhost');
-define ('STORAGE_DB', 'poche');
-define ('STORAGE_USER', 'poche');
-define ('STORAGE_PASSWORD', 'poche');
+define ('STORAGE_DB', 'wallabag');
+define ('STORAGE_USER', 'wallabag');
+define ('STORAGE_PASSWORD', '');
 
 #################################################################################
 # Do not trespass unless you know what you are doing
